lms/djangoapps/bookmarks/tests/test_views.py

```diff
@@ -312,6 +359,28 @@ def test_unsupported_methods(self):
         self.assertEqual(405, self.client.put(reverse('bookmarks')).status_code)
         self.assertEqual(405, self.client.delete(reverse('bookmarks')).status_code)
 
+    @ddt.data(
+        (-1, (2, 10, 1)),
+        (0, (2, 10, 1)),
+        (999, (2, 500, 1)),
+    )
+    @ddt.unpack
+    @patch('lms.djangoapps.bookmarks.views.tracker.emit')
+    def test_listed_event_for_different_page_size_values(self, page_size, expected_values, mock_tracker):
```

Comment:
You can use variables like `expected_bookmarks_count` etc, instead of putting the values in a tuple.